frontend/webpack.config.ts

```diff
@@ -55,7 +55,8 @@ const config: webpack.Configuration = {
       },
       {
         test: /\.s?css$/,
-        exclude: /node_modules/,
+        // PF loads CSS directly from node_modules\@patternfly now so removing this
```

Comment:
this may be required for react-core `3.2.4` update

Reply:
resolved in #1546 see: https://github.com/openshift/console/pull/1546#discussion_r282474811